Source/WebCore/dom/Element.cpp

```diff
@@ -3794,7 +3799,7 @@ ExceptionOr<void> Element::setInnerHTML(const String& html)
 String Element::innerText()
 {
     // We need to update layout, since plainText uses line boxes in the render tree.
-    document().updateLayoutIgnorePendingStylesheets();
+    document().updateLayoutForNode(*this);
```

Comment:
Do we need this for innerText() ? Since skipped content is skipped from innerText, I'm not sure we need to lay it out (I may be wrong though)

Reply:
Indeed it is not needed, the important thing is to make sure updateRelevancyOfContentVisibilityElements is called and style is up-to-date, fixed.